indicators/McAfee-Virus-8wer3hwa.yml

```diff
@@ -0,0 +1,33 @@
+title: McAfee Virus Phishing Kit 8wer3hwa
+description: Detects phishing pages mimicking McAfee with suspicious behavior.
+level: likely_malicous
+    
+references:

```

Comment:
Definitely needs references.

Reply:
I don't have any references of a mcafee scam site.